src/content/blog/2024_zne_catalyst.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Just in time for just-in-time error mitigation"
+author: Alessandro Cosentino
+day: 31
+month: 10
+year: 2024
+---
+
+We all want faster computer programs. Just-in-time (JIT) compilation has been a key concept in classical compiler design to help achieve faster computer programs. Most modern programming languages and runtime environments support some kind of JIT optimization techniques. In fact, very recently, an [experimental JIT compiler](https://docs.python.org/3/whatsnew/3.13.html#whatsnew313-jit-compiler) was added to Python in its latest release, 3.13.
+
```

Comment:
minor, but it could be worth (very briefly!) explaining what a JIT is, as readers might not be familiar (could be as simple as saying, at least in this context 'compiling parts of your workflow for performance improvements at execution time, avoiding sometimes slow Python interpretation'.